VargasMartinezJose/Dis45/HackerRank/ejerciciojava.java/JavaIfElse.java

```diff
@@ -0,0 +1,20 @@
+public class JavaIfElse {
+
+    private static final Scanner scanner = new Scanner(System.in);
+
+    public static void main(String[] args) {
+
+        int number = scanner.nextInt();
+        scanner.skip("(\r\n|[\n\r\u2028\u2029\u0085])?");
+
+        if (number % 2 == 1 | (number % 2 == 0 & number >= 6 && number <= 20)) {
```

Comment:
operator | ? &?
I think you mean || and && right?

Do you really need all conditionals to be evaluated??? 
Remember difference between & and &&